### PR TITLE
Updating orchestrator and validator jobs with service bus metrics changes

### PR DESCRIPTION
**Pull request by agr**
Change is verified in dev.

### CODE REVIEW

File: src/NuGet.Services.Validation.Orchestrator/Telemetry/TelemetryService.cs [hunk 1]
```diff
@@ -221,5 +225,23 @@ public IDisposable TrackDurationToStartPackageSigningValidator()
...
         public IDisposable TrackDurationToStartPackageCertificatesValidator()
             => _telemetryClient.TrackDuration(DurationToStartPackageCertificatesValidatorSeconds);
+
+        public void TrackMessageDeliveryLag<TMessage>(TimeSpan deliveryLag)
+            => _telemetryClient.TrackMetric(
+                MessageDeliveryLag,
+                deliveryLag.TotalSeconds,
+                new Dictionary<string, string>
+                {
+                    { MessageType, typeof(TMessage).Name }
```
Comment (loic-sharma) [inline src/NuGet.Services.Validation.Orchestrator/Telemetry/TelemetryService.cs]: Does `nameof(TMessage)` work by any chance?

File: src/Validation.PackageSigning.ValidateCertificate/TelemetryService.cs [hunk 2]
```diff
@@ -4,21 +4,26 @@
...
 
 namespace Validation.PackageSigning.ValidateCertificate
 {
-    public class TelemetryService : ITelemetryService
+    public class TelemetryService : ITelemetryService, ISubscriptionProcessorTelemetryService
     {
         private const string PackageSignatureMayBeInvalidated = "PackageSignatureMayBeInvalidated";
         private const string PackageSignatureShouldBeInvalidated = "PackageSignatureShouldBeInvalidated";
         private const string UnableToValidateCertificate = "UnableToValidateCertificate";
+        private const string Prefix = "ValidateCertificate.";
```
Comment (loic-sharma) [inline src/Validation.PackageSigning.ValidateCertificate/TelemetryService.cs]: Oops, I forgot the prefix - good catch :)

Could you add the prefix to all the other metric names as well?

File: src/Validation.PackageSigning.ValidateCertificate/TelemetryService.cs [hunk 3]
```diff
@@ -64,5 +69,23 @@ public void TrackUnableToValidateCertificateEvent(EndCertificate certificate)
                     { CertificateThumbprint, certificate.Thumbprint }
                 });
         }
+
+        public void TrackMessageDeliveryLag<TMessage>(TimeSpan deliveryLag)
```
Comment (cristinamanum) [inline src/Validation.PackageSigning.ValidateCertificate/TelemetryService.cs]: Are these methods used anywhere?

Comment (agr) [reply src/Validation.PackageSigning.ValidateCertificate/TelemetryService.cs]: Yes, they are used by service bus library, changes were added in this PR: https://github.com/NuGet/ServerCommon/pull/164

File: src/NuGet.Services.Validation.Orchestrator/Telemetry/TelemetryService.cs (hunk 1 above)
Comment (agr) [reply src/NuGet.Services.Validation.Orchestrator/Telemetry/TelemetryService.cs]: It works, but produces `"TMessage"` :-)

**Pull request merged**